Frontend/src/components/MentorProfileModal.jsx

```diff
@@ -0,0 +1,151 @@
+import {useContext, useEffect, useState} from 'react'
+import { UserContext } from '../UserContext.jsx';
+import { useNavigate } from 'react-router-dom';
+import './MentorProfileModal.css'
+
+const experienceMappingReverse = {
+    1: '0-2',
+    2: '2-5',
+    3: '5-10',
+    4: '10+',
+    5: '20+',
+  };
+  
+
+function MentorProfileModal ({handleCheckboxChange, handleDropdownToggle, dropdownOpen, selectedSkills, skillsList, closeModal}) {
+    const { user } = useContext(UserContext);
+    const [formData, setFormData] = useState({
+        industry: '',
+        company: '',
+        work_role: '',
+        years_experience: '',
+        school: '',
+        skills: selectedSkills.join(', '),
+    })
+    const handleChange = (e) => {
+        const { name, value } = e.target;
+        setFormData({ ...formData, [name]: value });
+    };
+
+      const handleSubmit = async (e) => {
+        e.preventDefault();
+    
+        const preparedData = {
+          ...formData,
+          skills: selectedSkills.join(', '),
+        };
+    
+        try {
+          const response = await fetch(`http://localhost:3000/mentors/${user.id}`, {
```

Comment:
This should go into a const, or better yet a config file. localhost will only ever be correct for local development. By putting this into a config file, you can create a system that can automatically detect what environment is being run (developement, production etc) and choose the correct url

Reply:
Resolved